app-frontend/packs/components/dashboard/homePage/HomePageWithBanners.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+
+    <section class="tt-home-page-with-banners">
+
+        <UiButton class="tt-page__tt-button tt-page__tt-button--first"
+                  :typeButton="'button'"
+                  @click="createNewBanner">Create new banner</UiButton>
```

Comment:
если ты начал делать мультилайн, то уже все проперти нужно мультилайном делать
```suggestion
        <UiButton 
            class="tt-page__tt-button tt-page__tt-button--first"
            :typeButton="'button'"
            @click="createNewBanner">
                 Create new banner
             </UiButton>
```

Reply:
понял, щас всё исправлю